sip-tbd/sip-tbd-taproot_payments.md

```diff
@@ -0,0 +1,75 @@
+# Preamble
+
+SIP Number: [TBD]
+
+Title: Taproot Payments Integration
+
+Authors: Rena Shah ([rena@trustmachines.co](mailto:rena@trustmachines.co))
+
+Consideration: Governance, Technical
+
+Type: Consensus
+
+Status: Draft Proposal
+
+Created: 24 May 2022
+
+License: CC0-1.0
+
+Sign-off: Technical Committee
+
+# Abstract
+
+This SIP proposes a small but important change to enable mining payouts to Taproot scripts. Taproot scripts were enabled at block 709,632 on the Bitcoin Network on November 14, 2021. 
+
+We believe that with this change, mining pools are more realistic where people can pool their mining bids into a Taproot script while running independent miners. Doing so further increases the number of independent miners and aids decentralization. Taproot scripts alleviate bandwidth issues on the Bitcoin network since the pool resembles a single miner. Furthermore, we’d like to implement the optionality to send Taproot payments scripts to both Stacks wallets or Stacks smart contracts to streamline operations. 
+
+# License and Copyright
+
+This SIP is made available under the terms of the Creative Commons CC0 1.0 Universal license, available at [Creative Commons — CC0 1.0 Universal](https://creativecommons.org/publicdomain/zero/1.0/). This SIP’s copyright is held by the Stacks Open Internet Foundation.
+
+# Introduction
+The Taproot upgrade was first proposed to the Bitcoin network in 2018 to improve efficiency. As a result, one of the core updates was a technique called “key aggregation,” which enables multi-signature transactions to commit the same amount of data to the Bitcoin blockchain as a single standard transaction. With this technique, transactions are batched into a single transaction, resulting in a lighter load. 
+
+With this technology, there are benefits to the Stacks ecosystem with respect to mining. Enabling Taproot scripts allows independent miners to pool together their mining bids as a single transaction. With that change, more independent miners can come on board to pool together for block bids. Adding the optionality to send Taproot payments to Stacks smart contracts streamlines mining pools operations within their protocol. 
+
+# Modification
+
+This SIP proposes to integrate Taproot script payments to the Stacks blockchain. Additionally, this SIP also proposes to allow the option to send Taproot script payments to Stacks Smart Contracts.
+
+# Rationale
```

Comment:
This should be a 2nd level header or lower.  SIP-000 describes the 1st-level headers.